wchamber/opendj/kustomization.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+commonLabels:
+  env: wchamber
+namespace: wchamber
+patchesStrategicMerge:
+- stateful-set.yaml
+resources:
+# Will get overwritten with correct version on deploy:
+- github.com/databiosphere/terra-config//base-configs/opendj?ref=0.0.0
```

This won't work since this repo is not versioned like the service repos. You can just give a relative path.
```suggestion
- ../../base-configs/opendj
```